byon/files/seaclouds-catalog.bom

```diff
@@ -184,7 +184,7 @@ brooklyn.catalog:
         onExistingProperties: do_not_use
         brooklynnode.webconsole.nosecurity: true
         brooklynnode.classpath:
-          - https://s3-eu-west-1.amazonaws.com/seaclouds-deployer/deployer-0.9.0-20161101.16.20.24-46.jar
+          - https://s3-eu-west-1.amazonaws.com/seaclouds-deployer/deployer-0.9.0-20162302.17.14.52.jar
```

Comment:
`deployer-0.9.0-20162302.17.14.52.jar` is produced by SeaClouds/deployer

`brooklyn-tosca-transformer-0.9.0-SNAPSHOT-20162401.225903.jar` is produced from my brooklyn-tosca fork, [kiuby88/brooklyn-tosca integration/seaclouds-integration branch](https://github.com/kiuby88/brooklyn-tosca/tree/integration/seaclouds-integration).

This branch produces a uber jar for brooklyn-tosca/brooklyn-tosca-transformer module. This jar is added. Then, this jar is added to brooklyn deployer.